DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -63,6 +77,27 @@ public class DcContext {
         return DcUtils.copyAndFreeArray(inputArray: cContacts)
     }
 
+    public func getContact(id: Int) -> DcContact {
+        let contactPointer = dc_get_contact(contextPointer, UInt32(id))
+        return DcContact(contactPointer: contactPointer)
+    }
+
+    public func blockContact(id: Int) {
+        dc_block_contact(contextPointer, UInt32(id), 1)
+    }
+
+    public func unblockContact(id: Int) {
+        dc_block_contact(contextPointer, UInt32(id), 0)
+    }
+
+/*    public func getFromContact(messageId: Int): DcContact? {
+        DcContact(id: fromContactId)
+    }() */
+
+    public func isFromCurrentSender(message: DcMsg) -> Bool {
+        return message.fromContactId == getContact(id: Int(DC_CONTACT_ID_SELF)).id
```

Comment:
as `getContact(id: Int(DC_CONTACT_ID_SELF)).id` is always just  DC_CONTACT_ID_SELF (that was wrong before), there is no need to move that to `DcMsg` to `DcContext`.

nb: on android, we call this function `isOutgoing()` which is a better name, however, to keep diff small, we should not rename the function.